addons/binding/org.openhab.binding.network.test/src/test/java/org/openhab/binding/network/discovery/DiscoveryTest.java

```diff
@@ -24,6 +24,7 @@
 import org.mockito.Mock;
 import org.openhab.binding.network.NetworkBindingConstants;
 import org.openhab.binding.network.internal.PresenceDetectionValue;
+import org.openhab.binding.network.internal.discovery.NetworkDiscoveryService;
```

Comment:
I think the test class should be moved as well, so that this import should not be needed.